conjur/argument_parser/_screen_options_parser.py

```diff
@@ -0,0 +1,25 @@
+from conjur.version import __version__
+from conjur.argument_parser.parser_utils import conjur_copyright
+
+
+class ScreenOptionsParser:
```

Comment:
i would call the class MainScreenParser or MainOptionParser
```suggestion
class MainOptionParser:
```

Reply:
what about MainScreenOptions?

Reply:
sounds good!